hathor/cli/run_node.py

```diff
@@ -211,8 +213,9 @@ def create_wallet():
                 self.log.warn('--rocksdb-storage is now implied, no need to specify it')
             cache_capacity = args.rocksdb_cache
             use_memory_indexes = not args.x_rocksdb_indexes
-            tx_storage = TransactionRocksDBStorage(path=args.data, with_index=(not args.cache),
-                                                   cache_capacity=cache_capacity,
+            rocksdb_storage = RocksDBStorage(path=args.data, cache_capacity=cache_capacity)
+            tx_storage = TransactionRocksDBStorage(rocksdb_storage=rocksdb_storage,
```

Comment:
This looks like such a trivial argument that I think it might even be better to just not pass it by keyword, `TransactionRocksDBStorage(rocksdb_storage, ...)` seems a little bit better for me. This also applies for a few other cases where the keyword is being used.